src/test/ui/attributes/issue-99647.rs

```diff
@@ -0,0 +1,16 @@
+// edition:2018
+// compile-flags: -Zcrate-attr=feature(generic_const_exprs)
```

Comment:
hm, this doesn't depend on the flag, right? can this be made into a `#![feature]` instead and moved into like consts/ (i think there's a folder in there for generic const exprs)

Reply:
Oh, you're right! I thought it depended on as the issue mentioned it, but code with the `#![feature]` attr indeed ICE'd on nightly-2022-07-23. Pushed a change: https://github.com/rust-lang/rust/pull/105994/commits/ba8d7cd3e193983dbfee757a83532043c10d4529